src/ApprenticeAan/SFA.DAS.ApprenticeAan.Api.UnitTests/Services/QueryStringParameterBuilderTests.cs

```diff
@@ -0,0 +1,227 @@
+﻿using AutoFixture.NUnit3;
+using FluentAssertions;
+using SFA.DAS.ApprenticeAan.Api.Models;
+using SFA.DAS.ApprenticeAan.Application.CalendarEvents.Queries.GetCalendarEvents;
+using SFA.DAS.ApprenticeAan.Application.Common;
+using SFA.DAS.ApprenticeAan.Application.Services;
+
+namespace SFA.DAS.ApprenticeAan.Api.UnitTests.Services;
+
+public class QueryStringParameterBuilderTests
+{
+
+    [Test, AutoData]
+    public void Operator_PopulatesModelFromParameters(Guid memberId, DateTime? fromDate, DateTime? toDate,
+        List<EventFormat> eventFormats, List<int> calendarIds, List<int> regionIds, int? page, int? pageSize)
+    {
+        var model = new GetCalendarEventsRequestModel
```

Comment:
Can you use `sut` for the target object name please

Reply:
ok